websocketpp/impl/connection_impl.hpp

```diff
@@ -2336,6 +2340,9 @@ void connection<config>::log_fail_result()
     s << " " << m_ec << " " << m_ec.message();
 
     m_alog.write(log::alevel::fail,s.str());
+#ifdef AIRTIME_CHIOS_EXTRA_LOGGING
```

Comment:
I think alevel::fail is not filtered out (need to verify).

Reply:
do we want to? I left all the existing logs in place. alog also won't write to stderr, so we won't see it regardless.

Reply:
I meant for this particular one, it will be output, so might not be add ours.

Reply:
We want to output it specifically to error log though. because otherwise i will mess up the mocha test outputs.

Reply:
I see.

Reply:
in retrospect, this branch of code is probably not part of our failure mode, since if we did fail here, we would have seen the regular fail output corrupt our test output file.